src/storage/mod.rs

```diff
@@ -1073,6 +1199,73 @@ impl Storage {
             .flatten()
     }
 
+    pub fn async_raw_batch_scan(
```

Comment:
I don't think there is a such demand for batch scan, can you show me some use cases?